atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/sweep/CandidateRowsForSweepingIterator.java

```diff
@@ -20,29 +20,27 @@
 
 import com.google.common.collect.AbstractIterator;
 import com.google.common.collect.Iterables;
-import com.palantir.atlasdb.keyvalue.api.CandidateCellForSweeping;
 import com.palantir.atlasdb.keyvalue.api.CandidateCellForSweepingRequest;
-import com.palantir.atlasdb.keyvalue.api.KeyValueService;
 import com.palantir.atlasdb.keyvalue.api.RangeRequests;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
 import com.palantir.atlasdb.keyvalue.cassandra.CqlExecutor;
 import com.palantir.common.base.ClosableIterator;
 
-public class GetCandidateCellsForSweepingIterator extends AbstractIterator<List<CandidateCellForSweeping>>
-        implements ClosableIterator<List<CandidateCellForSweeping>> {
+public class CandidateRowsForSweepingIterator extends AbstractIterator<List<CandidateRowForSweeping>>
+        implements ClosableIterator<List<CandidateRowForSweeping>> {
 
-    private final KeyValueService kvs;
+    private final ValuesLoader valuesLoader;
     private final CqlExecutor cqlExecutor;
     private final TableReference table;
     private final CandidateCellForSweepingRequest request;
 
     byte[] nextStartRow;
 
-    public GetCandidateCellsForSweepingIterator(
-            KeyValueService kvs,
+    public CandidateRowsForSweepingIterator(
+            ValuesLoader valuesLoader,
             CqlExecutor cqlExecutor, TableReference table,
```

Comment:
nit: this formatting is weird - either one parameter per line, or as many parameters per line as baseline allows.

Intellij does this kind of thing sometimes when you make a refactor :(